apps/root/src/pages/home/components/portfolio/index.tsx

```diff
@@ -184,8 +192,14 @@ const PortfolioBodyItem: ItemContent<BalanceItem, Context> = (
               <Skeleton variant="text" animation="wave" sx={{ minWidth: '5ch' }} />
             ) : (
               <ContainerBox alignItems="center" gap={3}>
-                <CircularProgressWithBrackground thickness={8} size={SPACING(6)} value={relativeBalance} />
-                <StyledBodySmallLabelTypography>{relativeBalance.toFixed(0)}%</StyledBodySmallLabelTypography>
+                <CircularProgressWithBrackground
+                  thickness={8}
+                  size={SPACING(6)}
+                  value={showBalances ? relativeBalance : 0}
```

Comment:
This got me thinking... Should this % be hidden? I don't think it's wrong to hide it, but I also don't consider it sensitive information

Reply:
Nope, at least based on the designs we want to show it